features/confidential-transactions/index.md

```diff
@@ -90,13 +90,11 @@ elements-cli sendrawtransaction $TX
 
 The implementation of Confidential Transactions as it appears in Elements has some important limitations to be aware of.
 
-The implementation only hides a certain number of the digits of the amount of each transaction output, dependent on the range proof's "blinding coverage" at a desired precision level.  Subsequently, there is a "minimum confidential amount" around 0.0001, and a "maximum confidential amount" that is 2<sup>32</sup> times the minimum amount.
-
-Digits smaller than the minimum will be revealed to observers; for example, if the minimum is 0.0001, a transaction output of 123.456789 will look like "?.????89". The tiny amount of information about the value leaked in this way is unlikely to be important, but be aware that this could allow observers to "follow" coins by linking subsequent transactions with identical amounts. All values can be rounded to the minimum to avoid revealing information in this way if preferred.
+The implementation only hides a certain number of the digits of the amount of each transaction output, dependent on the range proof's "blinding coverage" at a desired precision level.  Subsequently, the default "maximum confidential amount" is just over 45 million (45,035,996.27370496). For assets that are pegged to Bitcoin, such as L-BTC on the Liquid network, it is impossible to exceed 2^52 satoshis. For other Issued Assets it may be possible to exceed the maximum confidential amount by repeatedly reissuing the asset and then spending an amount greater than the limit in a single transaction, and so the limitation should be noted.
```

Comment:
Is it actually limited by 52 bits, or this is only a default setting for Elements client ?
https://github.com/ElementsProject/elements/pull/834 allows to set `ct_bits` to any value, and only secp256k1-zkp library checks will be the limiting factor, AFAIU.

The check in secp256k1-zkp lib is for 64 bits, not 52 bits:
https://github.com/ElementsProject/secp256k1-zkp/blob/cad7cc8f34d17166da2dabae20188ea6e747ac69/src/modules/rangeproof/rangeproof_impl.h#L217

Reply:
Yeah Dmitri is correct here. I think the wording is a bit confusing. I think it would be better to just say that "the default blinding precision is 52-bits, which will cover any value up to 2^52 satoshis (for L-BTC this is 45 million)". It's not a maximum, but it's just the default (that both Elements Core and Green have, which I think is the vast majority of our users).